Documentation/commands.md

```diff
@@ -129,6 +129,28 @@ $ journalctl -M rkt-bc3c1451-2e81-45c6-aeb0-807db44e31b4 -t redis
 
 Additionaly, logs can be programmatically accessed via the [sd-journal API](https://www.freedesktop.org/software/systemd/man/sd-journal.html).
 
+Currently there are two known main issues with logging in rkt:
+* In some rare situations when an application inside the pod is writing to `/dev/stdout` and `/dev/stderr` (i.e. nginx) there is no way to obtain logs.
+ The app should be modified so it will write to `stdout` or `syslog`. In the case of nginx:
+ ```
+ error_log stderr;
+ 
+ http {
+     access_log syslog:server=unix:/dev/log main;
+     [...]
+ }
+ ```
+ should be added to ```/etc/nginx/nginx.conf```
+
+* If application inside the pod (i.e. newest etcd) writes directly to journald, it's resulting in its log entries being scoped by machine UUID.
```

Comment:
"Some applications, like newer version of etcd, write directly to journald. Such log entries will not be written to stdout or stderr."